social-messenger-ts/package.json

```diff
@@ -1,18 +1,24 @@
 {
-  "name": "team",
+  "name": "@stream-io/social-messenger-demo",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@stream-io/stream-chat-css": "^2.8.1",
+    "@stream-io/stream-chat-css": "^2.9.0",
```

Comment:
@MartinCupela @oliverlaz @szuperaz one thing which I lean towards is making the css package "internal" (We've discussed this with Zita) - the React build system copies the CSS files upon build and they can be referred from the `stream-chat-react` dist directory. 

This would prevent people from having accidental bugs because they did not upgrade the CSS package. Also, it will simplify the installation instructions.

Reply:
Hey @petyosi, this makes sense to me as well. We can have the required CSS files bundled with the React SDK and we'll have control over the loaded stylesheet. I'll put an item to analyze this in my to-do list.

Reply:
Hey @petyosi, this makes sense to me as well. We can have the required CSS files bundled with the React SDK and we'll have control over the loaded stylesheet. I'll put an item to analyze this in my to-do list.

Reply:
We already have them bundled, it's a step done so that we don't break the SDK when we extracted the stylesheets for the Angular SDK.